wordle-web/pages/index.vue

```diff
@@ -12,7 +12,7 @@
       <v-card >  
 
         <v-card-title class="headline">
-          Welcome to the Game
+          Welcome to Hewr and Ben Game
```

Comment:
I looked at your web app deployment, and everything seems to be working well. I like the MDI-token you used in the  page menu header.

Reply:
thanks you